content/source-git/status.md

```diff
@@ -0,0 +1,85 @@
+  * Critical fixes are being resolved to maintain the current feature set.
+  * The team will not implement any new functionality (but we encourage/help others to do it).
+  * The team will check the demand and the priority of this feature on a quarterly basis to update the priority of this effort.
+
+## FAQ
+
+### I really want this feature to be implemented. What can we do?
+You can definitely reach the team and we can discuss what needs to be done. We can either help you implement this or (in case of higher demand) Packit team will implement this for you. The Packit team will not pick any new work without strong justification.
+
+### Is Source Git dead?
+No. This is just about deprioritizing this effort. Packit team regularly checks the priorities and if the demand for this increases, the team will prioritise this again.
+
+### Can we do something to move this effort forward? 
+Yes, either bring multiple other teams and reach us, or we are willing to pass this to a new team to continue the work.
+
+### Can we do something to help this succeed in the future?
+Yes, simplify the distribution workflow itself and reduce differences between Fedora and RHEL/Stream. Ask the Red Hat “Operating System CI” (OSCI) team and/or [Veronika Kabatova](https://github.com/veruu) about how you can help.
+
+### Why don’t we remove dist git completely? Wouldn’t it be easier?
+It would be really nice, but it’s a bold move that needs to be planned and done properly to be accepted. We, as a Packit team, don’t have enough people and power to do this. This shouldn’t be a decision of the Packit team, but we can be asked to do the implementation once it’s agreed on. (Because of this, we went with an opt-in approach instead. Note the ratio of packages potentially interested in Source Git.)
+
+### Was the work on this a waste of time?
+We still think that the dist git approach needs to be improved to make the distribution work more efficient. We’ve learned a lot and collected some valuable data/feedback that can be reused. The developed solution still provides some value and can be built upon. The source-git work was done as part of the Packit project which is successful as a whole. Having it as an extra deployment made us do the project more modular.
+Also, to move forward as a team/company, we need to try new things, but we need to realise when to stop or slow down.
+
+### What could have been done better?
+Changing the core part of a development workflow, source code handling, is a huge task. We intended to do this in all three distributions: RHEL, Fedora Linux and CentOS Stream. That is a humongous scope, especially when the infrastructure behind these distros is changing. Having more reasonable expectations and smaller work items could result in more use of our source-git workflow. Now that gitlab.com is becoming the center piece for all sources, it will be more manageable to implement complex workflow changes.
```

Comment:
```suggestion
Changing the core part of a development workflow, source code handling, is a huge task. We intended to do this in all three distributions: RHEL, Fedora Linux and CentOS Stream. That is a humongous scope, especially when the infrastructure behind these distros is changing. Having more reasonable expectations and smaller work items could result in more use of our source-git workflow. Now that gitlab.com is becoming the centerpiece for all sources, it will be more manageable to implement complex workflow changes.
```